source/FunkinLua.hx

```diff
@@ -81,8 +81,8 @@ class FunkinLua {
 		#end
 
 		// Lua shit
-		set('Function_Stop', Function_Stop);
-		set('Function_Continue', Function_Continue);
+		set('Function_Stop', 'Function_Stop');
+		set('Function_Continue', 'Function_Continue');
```

Comment:
To get cutscenes to work properly on Mac. Although I might need someone to test if it will still work on Windows like this because I don't have a Windows.